src/start/worker.ts

```diff
@@ -36,6 +36,9 @@ main()
  */
 export async function main() {
   const { requestTraceHeader, adminPort, host } = getConfig()
+  mergeConfig({
+    pgQueueEnableWorkers: true,
+  })
```

Comment:
Doesn't this bypass config settings? Was this for debugging and left in by accident or intentional?